### ISSUE TITLE
Implement remaining integration runtime upgrade features.

### ISSUE THREAD

**Issue by lburgazzoli** [syndesisio/syndesis#3119]
## This is a...

<pre><code>
[X] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

We need to:
- raise a warning if the integration runtime runs on an older version
- include runtime version in integration
- generate modification notes
- etc

Implements  https://github.com/syndesisio/syndesis/issues/1128

**Comment by rhuss** [comment]
No. We "only" implemented the infrastructure operator for installing and updating the infrastructure components.

**Comment by lburgazzoli** [comment]
keep it open

**Comment by lburgazzoli** [comment]
keep it open

**Comment by rhuss** [comment]
@lburgazzoli fyi, if you like you can apply the label 'status/never-stale' in which case stale bot will shut up forever for a particular issue.
